operations/messagethreads.md

```diff
@@ -13,6 +13,7 @@ Get all message threads that you have created, filtered by time interval and/or
     "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
     "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
     "Client": "Sample Client 1.0.0",
+    "EnterpriseId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",

```

Comment:
Wait, why is this EntepriseId? You should be using IEnterpriseFilterParameters So there is EntepriseIds. 
cc @jonas-paul

Reply:
All getAll shoudl have EntepriseIds/ChainIds and only write endpoint have single Id. If it's like that in the code please fix it there as well, docs can be published meanwhile I guess.

Reply:
In code it is ok, so just fix it here.